mino/minogrpc/server_test.go

```diff
@@ -968,6 +1293,36 @@ func (m fakeMembership) Len() int {
 	return len(m.addrs)
 }
 
+// Handler:
+// implements a handler where the stream sends back the message with its id
+type testMeshHandler struct {
+	mino.UnsupportedHandler
+	addrID string
+}
+
+func (t testMeshHandler) Stream(out mino.Sender, in mino.Receiver) error {
```

Comment:
To make your test easier, you could have a wait group that is done when receiving a message so you don't have to manage the response in the test as you only want to verify that everyone is receiving the message.

Reply:
that's a solution. In this case my approach was that checking for a return is harder and tests more logic than only checking clients received the message.